### PR TITLE
Feature/recruitment header nav

### PR DESCRIPTION
**Pull request by jkellerman**
Closes #

## 📝 Description

- Created a new Header component called HeaderInner
- Added new HeaderInner component to recruitment website header component
- Added a new class modifier to Header component styles in osc-ui
- Changed background colour of navLink component
- Added a new utility class to utils
- Used new utility class to change the colour of bottom border for mobile navLinks

## ⛳️ Current behavior (updates)

- Huge gap between top of the webpage and Header, caused by there being to rows in grid

## 🚀 New behavior

- Gap no longer showing after refactoring grid to one row.

## 💣 Is this a breaking change (Yes/No):

Affects the header for ecommerce site with new HeaderInner component

### CODE REVIEW

File: packages/osc-ui/src/components/Header/Header.tsx [hunk 1]
```diff
@@ -63,13 +63,32 @@ export const Header = forwardRef<ElementRef<'header'>, HeaderProps>((props, forw
...
 });
 Header.displayName = 'Header';
+/* -------------------------------------------------------------------------------------------------
+ * HeaderInner
+ * -----------------------------------------------------------------------------------------------*/
+export interface HeaderInnerProps extends SharedNavProps {
+    /**
+     * Sets the header to be sticky
+     * @default false
+     */
```
Comment (stevenbister) [inline packages/osc-ui/src/components/Header/Header.tsx:78]: Probably don't need this comment?

**Review by stevenbister — CHANGES_REQUESTED** [review]
Looks good! One comment on some jsdoc that probably can be removed :)

I did also notice this behaviour?

https://github.com/Open-Study-College/osc/assets/23461173/9fc5f64d-65da-4c3a-8a09-9fe755e8f8dc


Where the header disappears after a certain distance scrolled? Is that intentional? It doesn't do it on the ecommerce site

File: packages/osc-ui/src/components/Header/Header.tsx (hunk 1 above)
Comment (jkellerman) [reply packages/osc-ui/src/components/Header/Header.tsx:78]: I didn't intentionally make any changes to scroll behavior, I've had a look at the code and I can't seem to spot why it this is happening. All I did was copy over the nav, inserted the new component and removed the action nav. Will have another look tomorrow to see if I can spot what is happening.

**Review by stevenbister — COMMENTED** [review]
Looks really good mate! One last thing I think, now that the sticky stuff is working okay. 
Do you think it's worth shrinking the logo size on mobile? The header takes up a fair bit of space. Happy to include this in a more design focussed PR if you like though?

**Comment by jkellerman** [comment]
> Looks really good mate! One last thing I think, now that the sticky stuff is working okay. Do you think it's worth shrinking the logo size on mobile? The header takes up a fair bit of space. Happy to include this in a more design focussed PR if you like though?



> Looks really good mate! One last thing I think, now that the sticky stuff is working okay. Do you think it's worth shrinking the logo size on mobile? The header takes up a fair bit of space. Happy to include this in a more design focussed PR if you like though?

Yeah currently working on that on another feature branch so we're good to close this now.

**Review by stevenbister — APPROVED** [review]
LGTM

**Pull request merged**